### PR TITLE
Hotfix/2.4.2

### PR DESCRIPTION
**Pull request by jchen1005**

### CODE REVIEW

File: exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultExamineeProcessor.java [hunk 1]
```diff
@@ -103,11 +104,11 @@ public int parseSchoolId(final Examinee examinee) throws ImportException {
...
     @Override
-    public Student parseStudent(final Examinee examinee, final int schoolYear, final int schoolId) throws ImportException {
-        final DataElementErrorCollector errorCollector = new DataElementErrorCollector();
-        final ExamParserHelper parserHelper = new ExamParserHelper(errorCollector, optionalDataElements.getOptionalDataElements());
+    public Student parseStudent(final Examinee examinee, final Exam exam, final int schoolId, DataElementErrorCollector examErrorCollector) throws ImportException {
+        final ExamParserHelper parserHelper = new ExamParserHelper(examErrorCollector, optionalDataElements.getOptionalDataElements());
+        final DataElementErrorCollector studentExamErrorCollector = parserHelper.getElementErrorCollector();
         if (examinee.getIsDemo()) {
-            errorCollector.add(new DataElementError("isDemo", "true", "demo students are not supported"));
+            studentExamErrorCollector.add(new DataElementError("isDemo", "true", "demo students are not supported"));
```
Comment (norway77) [inline exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultExamineeProcessor.java:111]: Why is this here

Comment (gregcharles) [reply exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultExamineeProcessor.java:111]: I don't know, but the special demo handling dates back to at least 6/15/18. Junyu combined the collection of student and exam validation errors, but didn't otherwise change the logic.
